nexus/swags/models.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# Third Party Stuff
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+# nexus Stuff
+from nexus.base.models import ImageMixin, TimeStampedUUIDModel
+
+
+class Swag(ImageMixin, TimeStampedUUIDModel):
+    item = models.ForeignKey('SwagItem', on_delete=models.CASCADE, verbose_name=_('Item'), null=False, blank=False)
+    users = models.ManyToManyField('users.User', through='SwagOwnership')
+    description = models.TextField(verbose_name=_('Description'), null=False, blank=False)
+
+    class Meta:
+        verbose_name = _('Swag')
+        verbose_name_plural = _('Swags')
```

Comment:
`verbose_name` and `verbose_name_plural` should be in lowercase.

Reply:
Since these are for displaying the relevant names in Admin, title case is good enough for them. Also with translations enabled, they will be displayed differently in admin according to the locale of the user.

Reply:
Yes, that makes sense. Then we also need to fix the [coding_rules.md](https://github.com/CuriousLearner/nexus/blob/master/docs/backend/coding_rules.md) regarding these should be in `Title case`.